docs/source/notebooks/power_analysis.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>Add the following in order to make tables more visible/simpler in the rendered docs</p><p> </p><p>pd.set_option('display.precision', 2)</p><p> </p><p>Or even a precision of 1... see what you think works well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/277/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>